gitoid/Cargo.toml

```diff
@@ -36,6 +36,8 @@ serde = { version = "1.0.197", optional = true }
 tokio = { version = "1.36.0", features = ["io-util"], optional = true }
 url = { version = "2.4.1", optional = true }
 boring = { version = "4.6.0", optional = true }
+openssl = {version = "0.10.66", optional = true }
+openssl-sys = { version = "0.9.103" , optional = true}
```

Comment:
I just checked to confirm, and directly importing `openssl-sys` isn't necessary. You can remove that. Otherwise, LGTM!

Reply:
Should be removed.